style.css

```diff
@@ -415,6 +420,65 @@ hr.mobile {
   display: none;
 }
 
+.modal-mobile,
+.modal-desktop {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  z-index: 999;
+  padding: 5px 12px 24px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  width: 100%;
+
+  /* box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px; */
```

Comment:
- [ ] Please remove commented code from the file, It's not good practice to leave commented code in the files.